cmd/helm/helm.go

```diff
@@ -75,6 +65,11 @@ Environment:
   $HELM_NO_PLUGINS    disable plugins. Set HELM_NO_PLUGINS=1 to disable plugins.
   $TILLER_NAMESPACE   set an alternative Tiller namespace (default "kube-system")
   $KUBECONFIG         set an alternative Kubernetes configuration file (default "~/.kube/config")
+  $HELM_TLS_CA_CERT   path to TLS CA certificate used to verify the Helm client and Tiller server certificates (default "$HELM_HOME/ca.pem")
```

Comment:
maybe error here: comment says that `ca.pem` is default but there is no other (not deleted) `ca.pem` occurrences in this patch.

Reply:
The default values are set in pkg/helm/helmpath/helmhome.go and the Init function in pkg/helm/environment/environment.go has been changed to use these - see "TLS defaults that depend on Home value".